enterprise/tpa_pipeline.py

```diff
@@ -103,7 +103,7 @@ def get_user_from_social_auth(tpa_provider, user_id):
         user_id (str): User id of user in third party LMS
 
     """
-    provider_slug = tpa_provider.provider_id.strip('saml-')
+    provider_slug = tpa_provider.provider_id[5:]
```

Comment:
Given this makes the intent less obvious and feels like a magic number, can you put a comment indicating what the 5 characters we're chopping off are.